### PR TITLE
feat: env var for custom order-book API urls

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

The order-book API endpoint is configured using the environment variable `REACT_APP_ORDER_BOOK_URLS`

### CODE REVIEW

**Review by alfetopito — COMMENTED** [review]
Have you tested passing the env var via CLI for example?

I would expect any env var needs to be added to webpack config like https://github.com/cowprotocol/explorer/blob/b6493bd1b4ca93088074e28893eecb75108f6ccf/webpack.config.js#L30-L35

Also, should be safe to remove these env vars and all the associated code which is no longer in use https://github.com/cowprotocol/explorer/blob/6b3c77dee2fa30a068555e1cc31b8b44c6dd8f2c/src/api/operator/operatorApi.ts#L12-L30

Lastly, is it possible to pass to the SDK (like the settings above) prod & barn api endpoints?

**Comment by shoom3301** [comment]
@alfetopito thank you!
Added the var into webpack config, tested, it works.

>is it possible to pass to the SDK (like the settings above) prod & barn api endpoints?
No, but I understand why you asking. We use `getOrderMultiEnv` in Explorer.
I will try to add a configuration for this case in the next PR.

**Review by alfetopito — COMMENTED** [review]
One more ask: Can you also remove this dead code that is no longer in use and are related to the current changes?

https://github.com/cowprotocol/explorer/blob/b6493bd1b4ca93088074e28893eecb75108f6ccf/webpack.config.js#L30-L35
https://github.com/cowprotocol/explorer/blob/6b3c77dee2fa30a068555e1cc31b8b44c6dd8f2c/src/api/operator/operatorApi.ts#L12-L30

File: .env.example [hunk 1]
```diff
@@ -10,4 +10,7 @@ REACT_APP_IPFS_READ_URI=https://cloudflare-ipfs.com/ipfs
 # Sentry
 #REACT_APP_SENTRY_DSN='https://<url>'
 #REACT_APP_SENTRY_TRACES_SAMPLE_RATE="1.0"
-#REACT_APP_SENTRY_AUTH_TOKEN='<sentry_auth_token>'
\ No newline at end of file
+#REACT_APP_SENTRY_AUTH_TOKEN='<sentry_auth_token>'
+
+# Orderbook API Endpoints
+#REACT_APP_ORDER_BOOK_URLS='{"1":"https://YOUR_HOST","100":"https://YOUR_HOST","5":"https://YOUR_HOST"}
```
Comment (anxolin) [inline .env.example:16]: good! since its coherent with CoW Swap one :)

**Review by anxolin — APPROVED** [review]

**Comment by shoom3301** [comment]
@alfetopito sure! Remove the unused code

**Review by alfetopito — APPROVED** [review]
Thank you!!!

**Comment by shoom3301** [comment]
https://github.com/cowprotocol/cowswap/pull/3584
The PR was migrated to the new repo

**Pull request closed without merging**